src/promptflow/promptflow/_sdk/entities/_trace.py

```diff
@@ -124,6 +131,16 @@ def _from_protobuf_object(obj: PBSpan, resource: typing.Dict) -> "Span":
             SpanStatusFieldName.STATUS_CODE: parse_otel_span_status_code(obj.status.code),
         }
         attributes = flatten_pb_attributes(span_dict[SpanFieldName.ATTRIBUTES])
+        if SpanAttributeFieldName.SESSION_ID not in attributes:
```

Comment:
why not write this in two if else. Current logic assumes session_id & span_type should coexist.

Reply:
good catch, let me refine this